changelogs/current.yaml

```diff
@@ -6,6 +6,14 @@ behavior_changes:
   change: |
     Removed support for (long deprecated) opentracing.  See `issue 27401
     <https://github.com/envoyproxy/envoy/issues/27401>`_ for details.
+- area: access_log
+  change: |
+    Added new JSON access log formatter implementation which provides 16-25x performance improvement over the old
+    implementation. The old implementation is deprecated and will be removed in a future release.
+    The :ref:`sort_properties <envoy_v3_api_field_config.core.v3.JsonFormatOptions.sort_properties>` field will
+    be ignored in the new implementation because the new implementation always sorts properties.
+    The behavior change can be reverted by setting the runtime guard
+    ``envoy.reloadable_features.logging_with_fast_json_formatter`` to false.
```

Comment:
I think except there is a break behavior change, it would be better to enable new feature by default. But I have no strong insistence on this point.

I think if we can add more tests to ensure the new formatter could output same json result as the old one in various cases. Then enable the new formatter would be a better choice. (But I have no idea about how to do that, will a fuzz test help?)

Reply:
Fuzz testing would be great.

The reason to have it off by default initially is that you will be able to battle-test in your production environment first, before all other Envoy deployments get it.

This is a really important principle. See https://www.crowdstrike.com/blog/falcon-content-update-preliminary-post-incident-report/  for what happens when this is not applied.

Reply:
I am fine to update the default flag value to false.

Reply:
TY -- a small edit to the release note text will then be needed.

Reply:
+1 we default true for Envoy unless it's particular high risk (like the codec swaps) and merits internal testing first